openlibrary/components/BulkSearch/utils/classes.js

```diff
@@ -122,6 +123,49 @@ export class AiExtractor extends AbstractExtractor{
     }
 }
 
+export class TableExtractor extends AbstractExtractor{
+
+    name = 'table_extractor'
```

Comment:
Oh forgot to say! Add a `name` to the other subclasses as well.

Reply:
Resolved